src/platz1de/EasyEdit/convert/ItemConvertor.php

```diff
@@ -67,6 +72,21 @@ public static function load(): void
 			EditThread::getInstance()->getLogger()->error("Failed to parse conversion data, Item conversion is not available");
 			EditThread::getInstance()->getLogger()->debug($e->getMessage());
 		}
+		EditThread::getInstance()->sendOutput(new NbtConversionData($rawConversionMap));
```

Comment:
Use the same package as other ressource data here
(cache the conversion map in a static variable like in [BlockTagManager](https://github.com/platz1de/EasyEdit/blob/master/src/platz1de/EasyEdit/convert/BlockTagManager.php#L33-L37), this isn't optimal but the currently used solution :/)
You'll probably also need to change the [loading order](https://github.com/platz1de/EasyEdit/blob/master/src/platz1de/EasyEdit/utils/ConfigManager.php#L256) so the item convertor is loaded before the block state convertor